tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-card.html

```diff
@@ -0,0 +1,306 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../paper-spinner/paper-spinner-lite.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-backend/tf-backend.html">
+<link rel="import" href="../tf-card-heading/tf-card-heading.html">
+<link rel="import" href="../tf-color-scale/tf-color-scale.html">
+<link rel="import" href="../tf-imports/lodash.html">
+<link rel="import" href="../vz-line-chart/vz-line-chart.html">
+
+<!--
+  Renders precision–recall curves.
+-->
+<dom-module id="tf-pr-curve-card">
+  <template>
+    <tf-card-heading tag="[[tag]]">
+    </tf-card-heading>
+
+    <div id="chart-and-spinner-container">
+      <vz-line-chart
+        x-components-creation-method="[[_xComponentsCreationMethod]]"
+        y-value-accessor="[[_yValueAccessor]]"
+        tooltip-columns="[[_tooltipColumns]]"
+        color-scale="[[_colorScaleFunction]]"
+      ></vz-line-chart>
+      <template is="dom-if" if="[[_loading]]">
```

Comment:
This is dead code: `_loading` is never defined. You can remove this element. (Then, `chart-and-spinner-container` should be renamed.)